ZConfig/cmdline.py

```diff
@@ -142,7 +174,10 @@ def _normalize_case(self, string):
         return string.lower()
 
 
-class MatcherMixin:
+class MatcherMixin: # pylint:disable=old-style-class
+    # Can't extend object without getting __init__ errors due to
```

Comment:
I can take a look this weekend if you don't beat me to it.

I've very I very much dislike having lint-control turds in code.